### DISCUSSION TITLE
connect to monailabel server

### DISCUSSION THREAD

**Discussion by muzahem** [Project-MONAI/MONAILabel#602] [Q&A] [answered]
I wanted to try MONAILabel in 3d Slicer on a sample data. But I cant connect to MONAILabel server. When I fill in the MONAI Label Server field with the URL http://127.0.0.1:8000/ it shows me this problem. 

![SharedScreenshot](https://user-images.githubusercontent.com/98332988/151934224-46584ac4-9035-4163-a471-4aeffbea6921.jpg)

**Comment by diazandr3s** [comment, marked as answer]
Hi @muzahem,

Thanks for opening this thread.
The URL you're providing to 3DSlicer means you're running the server locally. Have you installed MONAI Label on your computer? 
Please let us know if you've followed these instructions to start the server: https://github.com/Project-MONAI/MONAILabel/blob/main/README.md
